.github/workflows/test.yml

```
@@ -0,0 +1,53 @@
+name: test CI
+
+on: [pull_request]
+
+jobs:
+  pytest_with_coverage:
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v2
```

It is better to have a name:
```suggestion
      - name: checkout
        uses: actions/checkout@v2
```

---

```suggestion
        uses: actions/checkout@v2
```